Directory.Packages.props

```diff
@@ -76,7 +76,7 @@
   <ItemGroup Label="Production other packages">
     <PackageVersion Include="Amazon.Lambda.APIGatewayEvents" Version="[2.7.0,)" />
     <PackageVersion Include="Amazon.Lambda.ApplicationLoadBalancerEvents" Version="[2.2.0,)" />
-    <PackageVersion Include="Amazon.Lambda.Core" Version="[2.2.0,)" />
+    <PackageVersion Include="Amazon.Lambda.Core" Version="[2.8.0,)" />
```

Comment:
This is quite a large step forward in terms of the minimum version (~2 years of changes).